database/README.md

```
@@ -0,0 +1,81 @@
+# Database 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)